pages/_app.js

```diff
@@ -1,10 +1,18 @@
+import { useState } from "react";
 import GlobalStyle from "../styles";
 
 export default function App({ Component, pageProps }) {
-	return (
-		<>
-			<GlobalStyle />
-			<Component {...pageProps} />
-		</>
-	);
+  //Hier States
+  const [productToBuy, setProduct] = useState();
```

Comment:
pls correct setting a useState 

```suggestion
  const [productToBuy, setProductToBuy] = useState();
```